src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  width: 100%;
+}
+
+.search-form--big {
+  height: 70px;
+}
+
+.search-form--small {
+  height: 42px;
+}
```

Comment:
why don't we add the height of the big form right to the `.search-form` selector? In that case we won't need two additional selectors